aries-site/src/examples/templates/dashboards/content/chartCards/ChargesByMonth.js

```diff
@@ -0,0 +1,148 @@
+import { useEffect, useState } from 'react';
+import PropTypes from 'prop-types';
+import { Grid, DataChart, Text } from 'grommet';
+import { ChartCard, Measure } from '../../components';
+import { defaultWindow, formatCurrency, REPORT_WINDOW_MAP } from './utils';
+
+const MOCK_DATA = require('../../../../../data/mockData/consumption.json');
+
+export const ChargesByMonth = ({ period }) => {
```

Comment:
This seems like a 95% overlap of the CostByMonth. I am wondering if we should think about a more reusable approach here. Throwing out some ideas:

- Should we just consume the existing component from within the 4-col dashboard?
- Should we create a BarChartCard, which could be called by both ChargesByMonth and CostByMonth?
- Should we move the 'Set reporting window' and 'Filter and assemble data set' useEffects into their own reusable hooks (and migrate them to memos)?

Maybe we should make MonthlySpend the reusable component + make the hooks reusable?